_episodes/l2-03-releases_tags.md

```diff
@@ -163,15 +163,15 @@ Double-check that this is the commit you intended to tag by running `git log` (o
 `git graph`) again.
 
 Note that `tasty` can now be used like other git references, such as commit hashes and
-branch names. For example, you can run `git checkout tasty` to (temporarily) update the
-contents of your repo to be as they were back when you added half an onion to the
-instructions.
+branch names. For example, you can run `git switch --detach tasty` to (temporarily)
+update the contents of your repo to be as they were back when you added half an onion to
+the instructions. (Note that you need to include the `--detach` option!)
 
 You may now be wondering, if this is the case, then how is a tag different from a
 branch? Try checking out `tasty` to see what happens:
 
 ```
-git checkout tasty
+git switch --detach tasty
 ```
 {: .commands}
```

Comment:
Is the output below exactly correct?

Reply:
Good spot! No, actually `git` doesn't emit all that stuff about the detached `HEAD`, I guess because you have to opt in with the `--detach` flag. I'll sort it